src/main/kotlin/minesweeper/domain/Board.kt

```diff
@@ -1,11 +1,42 @@
 package minesweeper.domain
 
+import minesweeper.error.MineSweeperErrorMessage.OUTSIDE_THE_BOUNDS_OF_THE_BOARD
+
 class Board private constructor(
-    private val rows: Rows,
+    val rows: Rows,
+    private val minesCoordinates: Coordinates,
 ) {
+    private val height: Int
+        get() = rows.size
+
+    private val width: Int
+        get() = rows.first().size
+
```

Comment:
```suggestion
    private val height: Int = rows.size

    private val width: Int = rows.first().size
```

get을 써서 항상 불러올 때마다 가지고 오는 것보다는, 처음부터 주입하면 어떨까요?